resources/variants/gmax15plus_dual_025_e3d.inst.cfg

```diff
@@ -1,13 +0,0 @@
-[general]
```

Comment:
Similar as before, it's not a good idea to remove variants since then you need to also modify the VersionUpgrade to map the old variants to a new variant. If there is no other possibility, then please add this modification to your PR.

To better explain, imagine that you have added a gMax printer in Cura 3.5 and you selected this variant "0.25mm E3D (Difficult)". When upgrading to Cura 3.6 (for instance) you will get something like this:
![image](https://user-images.githubusercontent.com/15830521/47498500-bc72d300-d85d-11e8-99d3-2280d0ea51ab.png)

This is not desirable and we are not allowing this. You can fix it by modifying the Version Upgrade plugin by mapping the previous variant to a default variant. (Same if you rename the variant)

Reply:
We will add the variants back.